energy_calc/optimize_pulley.m

```diff
@@ -0,0 +1,109 @@
+% Sam Khalandovsky
+% Try to optimize the pulley parameters to decrease energy usage using
+% nonlinear optimization
+
+%% Inputs
+% traj: trajectory to optimize for
+% initial_pulley_vals: vector of parameters to optimize
+
+%% Outputs
+% pulley_vals: vector of optimized parameters
+% fraction_saved: fraction of energy saved by this pulley
+
+function [pulley_vals, fraction_saved] = optimize_pulley(traj, initial_pulley_vals)
+    % start with a candidate from knee_pulley
+    
+    % TODO constants separate
+    knee_motor_consts = [680, 14, .527];
```

Comment:
add comment that these are the motor values/ have me make one file where all these numbers are pulled from or something